gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/auth/user/impl/UserAuthenticationManagerImpl.java

```diff
@@ -150,6 +157,7 @@ public Single<User> authenticate(Client client, Authentication authentication, b
                 .doOnError(throwable -> {
                     gatewayMetricProvider.incrementFailedAuth(false);
                     eventManager.publishEvent(AuthenticationEvent.FAILURE, new AuthenticationDetails(authentication, domain, client, throwable));
+                    auditService.report(new AuthenticationAuditBuilder().client(client).throwable(throwable));
```

Comment:
**issue**: IMO This change is just duplicating the audit as currently the `USER_LOGIN` audit entry is managed by the `AuthenticationEventListener` which is listening the `AuthenticationEvent.FAILURE` published just before the `auditService.report` call you added. The change in the AuditBuilder should be enough to provide the info. wdyt ?

Reply:
When I was testing I got only one audit, It was no duplicated.